lib/actions.ex

```diff
@@ -207,6 +218,9 @@ defmodule ExTus.Actions do
         |> Utils.put_cors_headers()
         |> put_resp_header("Upload-Offset", "#{upload_info.offset}")
         |> put_resp_header("URL", "#{url}")
+        |> put_resp_header("HEIGHT", "#{height}")
```

Comment:
keep these new one in lowercase. DO NOT change URL to lowercase as it is already being used